rustual-boy-cli/Cargo.lock

```diff
@@ -267,7 +318,9 @@ dependencies = [
 
 [metadata]
 "checksum alsa-sys 0.1.0 (registry+https://github.com/rust-lang/crates.io-index)" = "2c3f84e271f1613d7d7cbf6608beaed341c3f3d7e57045a6fc1a7d112380613c"
+"checksum ansi_term 0.9.0 (registry+https://github.com/rust-lang/crates.io-index)" = "23ac7c30002a5accbf7e8987d0632fa6de155b7c3d39d0067317a391e00a2ef6"
```

Comment:
Since we've added dependencies here, we need to update the `LICENSE-THIRD-PARTY` file, which contains information for all of the dependencies we have when distributing binaries. Unfortunately, this needs to be done by hand (basically go to each crate's repo and reproduce the license [preferably the MIT license if possible] in the `LICENSE-THIRD-PARTY` file, along with the crate name and a link to said license file). I want us to do this whenever we change dependencies, as that's when we have most overview of why the dependencies are there and what was added, rather than at release time.

From what I can see here, we need to add license info for `ansi_term`, `clap`, `strsim`, `term_size`, `unicode-segmentation`, `unicode-width`, and `vec_map` (I may have missed one; I'm just reading the `Cargo.lock` diff here)